src/lib/components/events/event-carousel.svelte

```diff
@@ -24,13 +26,18 @@
 		event.preventDefault();
 		scrollTheCarousel(-event.deltaY);
 	};
+	onMount(() => {
+		hasHorizontalScrollbar = carouselRef.scrollWidth > carouselRef.clientWidth;
+	});
 </script>
 
 <section>
 	<div class="event-carousel-container">
-		<div bind:this={carouselButtonLeft} class="carousel-button left" on:click={scrollLeft}>
-			&lt;
-		</div>
+		{#if hasHorizontalScrollbar}
+			<div bind:this={carouselButtonLeft} class="carousel-button left" on:click={scrollLeft}>
+				&lt;
+			</div>
+		{/if}
```

Comment:
Instead of using Svelte's `{#if}` statement, I think it would make sense to use Svelte's [`class:*` directive](https://svelte.dev/docs#class_name) here. In the DIV that contains the left button, you can actually try adding the attribute below:

```html
<div class:enabled={leftButtonEnabled} />
```

This attribute makes sure that the element's class will be properly toggled depending on the boolean value of `leftButtonEnabled`.

Reply:
I added the attributed above instead of using the {#if} statement. I'm not sure if you wanted me to create two variables called leftButtonEnabled and rightButtonEnabled, I just used the hasHorizontalScrollbar variable.